### PR TITLE
LP 111 Apartment ViewModel creation

### PR DESCRIPTION
**Pull request by juanislas77**
Apartment ViewModel 
https://devaptivist.atlassian.net/browse/LP-111

Creating the ViewModel for the Apartment View

### CODE REVIEW

**Review by carlosph816 — APPROVED** [review]

File: feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt [hunk 1]
```diff
@@ -1,39 +1,71 @@
...
-fun ApartmentView(navigateToRepair : () -> Unit) {
-    ApartmentContainer { navigateToRepair() }
+fun ApartmentView(
+    navigateToRepair : () -> Unit,
+    viewModel: ApartmentViewModel = hiltViewModel()
+) {
+    LaunchedEffect(true){
+        viewModel.getInformation()
+    }
+    //ApartmentContainer { navigateToRepair() }
```
Comment (Lupillo17) [inline feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt]: Delete this commented line

File: feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt [hunk 2]
```diff
@@ -45,8 +77,20 @@ fun ApartmentContainer(navigateToRepair : () -> Unit) {
     ) {
         Column {
             TitleApartmentScreen()
-            PaymentsSection(Modifier, "Junary", "990.00", "Jan 23 2022")
-            RepairsSection { navigateToRepair.invoke() }
+            userData.paymentDetails?.currentMonth?.let {
+                userData.paymentDetails.deadline?.let { it1 ->
+                    PaymentsSection(
+                        modifier = Modifier,
+                        month = it,
```
Comment (Lupillo17) [inline feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt]: rename **it** for a readable name

File: feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt [hunk 3]
```diff
@@ -45,8 +77,20 @@ fun ApartmentContainer(navigateToRepair : () -> Unit) {
...
+                        quantityToPay = userData.paymentDetails.rentDebt.toString(),
+                        limitDateToPay = it1
```
Comment (Lupillo17) [inline feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt]: rename **it1** for a readable name

File: feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt [hunk 4]
```diff
@@ -45,8 +77,20 @@ fun ApartmentContainer(navigateToRepair : () -> Unit) {
...
+                    )
+                }
+            }
+            RepairsSection(
+                navigateToRepair ={ navigateToRepair.invoke() },
```
Comment (Lupillo17) [inline feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt]: you can change navigateToRepair.invoke() to navigateToRepair()

File: feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt (hunk 1 above)
Comment (juanislas77) [reply feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt]: Comment resolved

File: feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt (hunk 2 above)
Comment (juanislas77) [reply feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt]: Comment resolved

File: feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt (hunk 3 above)
Comment (juanislas77) [reply feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt]: Comment resolved

File: feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt (hunk 4 above)
Comment (juanislas77) [reply feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt]: Comment resolved

**Review by Lupillo17 — APPROVED** [review]

**Pull request merged**